utils/rpms/packaging/Dockerfile.mockbuild

```diff
@@ -29,6 +29,21 @@ RUN dnf -y install mock make                                        \
                    python-srpm-macros rpmdevtools &&                \
     dnf -y clean all
 
+# Install bullseye coverage
+ARG JENKINS_URL
+ARG BULLSEYE_KEY
+RUN set -x; if [ "x$BULLSEYE" != "x" ]; then                                                \
```

Comment:
Output from this change in https://github.com/daos-stack/daos/actions/runs/11243601101/job/31259882078?pr=15206
```
#6 [ 4/11] RUN set -x; if [ "x$BULLSEYE" != "x" ]; then                                                      dnf install xz && dnf clean all &&                                                  curl https://build.hpdd.intel.com/job/daos-stack/job/tools/job/master/lastSuccessfulBuild/artifact/bullseyecoverage-linux.tar         --retry 10 --retry-max-time 60 --silent --show-error -o bullseye.tar.xz &&        mkdir -p bullseye &&                                                                tar -C bullseye --strip-components=1 -xf bullseye.tar.xz &&                         pushd bullseye &&                                                                     ./install --quiet --key "***" --prefix /opt/BullseyeCoverage &&       popd &&                                                                             rm -rf bullseye.tar.xz bullseye;                                                  fi
#6 CACHED
```

Reply:
> Output from this change in [daos-stack/daos/actions/runs/11243601101/job/31259882078?pr=15206](https://github.com/daos-stack/daos/actions/runs/11243601101/job/31259882078?pr=15206)
> 
> ```
> #6 [ 4/11] RUN set -x; if [ "x$BULLSEYE" != "x" ]; then
>     dnf install xz && dnf clean all &&
>     curl https://build.hpdd.intel.com/job/daos-stack/job/tools/job/master/lastSuccessfulBuild/artifact/bullseyecoverage-linux.tar         --retry 10 --retry-max-time 60 --silent --show-error -o bullseye.tar.xz &&
>     mkdir -p bullseye &&
>     tar -C bullseye --strip-components=1 -xf bullseye.tar.xz &&
>     pushd bullseye &&
>     ./install --quiet --key "***" --prefix /opt/BullseyeCoverage &&
>     popd &&
>     rm -rf bullseye.tar.xz bullseye;
> fi
> #6 CACHED
> ```
> 
> So it does not appear to be doing anything.

Well, you don't know that because of the `CACHED`.  Some other run prior did the actual step and cached it.  You need to bust the cache there by adding `ARG CB0` before line 35.  That will force that step to be done on every run.

> Looking through the [daos-stack/daos/actions/runs/11243601101/artifacts/2031691676](https://github.com/daos-stack/daos/actions/runs/11243601101/artifacts/2031691676) logs, I do see - in root.log - that bullseye is installed via RPM by:

Indeed, as I mentioned [previously](https://github.com/daos-stack/daos/pull/15238#discussion_r1791936951).  I do believe that this is what is actually making all of this work and that the addition of https://github.com/daos-stack/daos/blob/20d013039686b87b9c4a8826ea1a5e3fb08d17a2/utils/rpms/packaging/Dockerfile.mockbuild#L32-L46 is not actually doing anything here.  You could just skip to the chase and remove that and see if things are still working here.